### PR TITLE
GEM - new gem emapping format for vfat v3

### PR DESCRIPTION
**Pull request by jshlee**
Splitting PR #25158 
This just adds in the new db format for vfat v3 of that original PR.

@tocheng @pohsun @kpedro88 @hyunyong @watson-ij

### CODE REVIEW

**Comment by tocheng** [comment]
@cmsbuild please test

**Comment by ggovi** [comment]
+1

**Comment by tocheng** [comment]
+1

File: CondFormats/GEMObjects/src/GEMeMap.cc [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      GEMROMapping::vfatEC ec;
+      ec.detId = gemId.chamberId();
+      ec.vfatAdd = imap.vfatAdd[ix] & chipIdMask_;
+      
+      GEMROMapping::vfatDC dc;
+      dc.vfatType = imap.vfatType[ix];
+      dc.detId = gemId;
+      dc.localPhi = imap.localPhi[ix];
+
+      std::cout << " vfatAdd " << ec.vfatAdd
```
Comment (fabiocos) [inline CondFormats/GEMObjects/src/GEMeMap.cc]: @jshlee @ggovi is this cout needed? This is not thread safe in case

Comment (ggovi) [reply CondFormats/GEMObjects/src/GEMeMap.cc]: @jshlee please remove the cout if not required, or replace them with the edm::Log functions

File: CondFormats/GEMObjects/src/GEMeMap.cc [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+      GEMROMapping::stripNum sMap;
+      sMap.vfatType = imap.vfatType[ix];
+      sMap.stNum = imap.vfatStrip[ix];
+
+      romap.add(cMap, sMap);
+      romap.add(sMap, cMap);
+    }
+  }
+  std::cout << " GEMeMap finished "
```
Comment (ggovi) [inline CondFormats/GEMObjects/src/GEMeMap.cc]: @jshlee same as above, please remove the std::cout

File: CondFormats/GEMObjects/src/GEMeMap.cc (hunk 1 above)
Comment (jshlee) [reply CondFormats/GEMObjects/src/GEMeMap.cc]: removed

File: CondFormats/GEMObjects/src/GEMeMap.cc (hunk 2 above)
Comment (jshlee) [reply CondFormats/GEMObjects/src/GEMeMap.cc]: removed

**Comment by fabiocos** [comment]
please test

**Comment by tocheng** [comment]
+1

**Comment by fabiocos** [comment]
@ggovi from your earlier approval only couts have been removed, so I expect you might confirm it easily

**Comment by ggovi** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Pull request merged**